addons/editor/initKeybinds.sqf

```diff
@@ -82,3 +82,60 @@
         };
     };
 }, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+[ELSTRING(main,DisplayName), QGVAR(watchCursor), [LSTRING(WatchCursor), LSTRING(WatchCursor_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+        curatorMouseOver params ["_type", "_entity", "_index"];
+        private _isCancelling = (_type == "OBJECT") && {_entity in SELECTED_OBJECTS};
+        private _gunners = (SELECTED_OBJECTS apply {gunner vehicle _x}) - [objNull];
+
+        private _cursorPosASL = call EFUNC(common,getCursorPosition);
+        {
+            [QEGVAR(common,doWatch), [_x, [ASLToAGL _cursorPosASL, objNull] select _isCancelling], _x] call CBA_fnc_targetEvent;
+        } forEach _gunners;
+
+        true // handled, prevents vanilla eject
```

Comment:
Just `handled` since unbound by default. Here and below.

Reply:
f2046a9